Samples/NetworkMediaStreamer/RTSPServer/build/RTSPServer.vcxproj

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|Win32">
+      <Configuration>Debug</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|Win32">
+      <Configuration>Release</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|x64">
+      <Configuration>Debug</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|x64">
+      <Configuration>Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="..\src\RTSPAuthProvider.cpp" />
+    <ClCompile Include="..\src\RTSPServer.cpp" />
+    <ClCompile Include="..\src\RtspSession.cpp" />
+    <ClCompile Include="..\src\SocketWrapper.cpp" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="..\..\Common\inc\RTSPServerControl.h" />
+    <ClInclude Include="..\inc\pch.h" />
+    <ClInclude Include="..\inc\RTSPServer.h" />
+    <ClInclude Include="..\inc\RtspSession.h" />
+    <ClInclude Include="..\inc\SocketWrapper.h" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\RTPMediaStreamer\build\RTPMediaStreamer.vcxproj">
+      <Project>{763269ff-37eb-4a0a-b152-0924cfa81fff}</Project>
+    </ProjectReference>
+  </ItemGroup>
+  <PropertyGroup Label="Globals">
+    <VCProjectVersion>16.0</VCProjectVersion>
+    <ProjectGuid>{1F92CAD2-33C6-42ED-AF2E-FDD0D8CE07BC}</ProjectGuid>
+    <Keyword>Win32Proj</Keyword>
+    <RootNamespace>VideoStreamer</RootNamespace>
+    <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
+    <ProjectName>RTSPServer</ProjectName>

```

Comment:
Look into ARM/ARM64, 18362 aspects.

Reply:
for ARM: -https://microsoft.visualstudio.com/_workitems/edit/29536275?fullScreen=True

---
In reply to: [491068201](https://github.com/microsoft/Windows-Camera/pull/13#discussion_r491068201) [](ancestors = 491068201)